docs/UserGuide.md

```diff
@@ -2,41 +2,125 @@
 
 ## Introduction
 
-{Give a product intro}
+ATHENA (Automated Timetable Helper Encourager n' Assistant) is a desktop daily life planner optimized for use via a Command Line Interface (CLI).
 
 ## Quick Start
 
 {Give steps to get started quickly}
 
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Download the latest version of `ATHENA` from [here](http://link.to/duke).
 
 ## Features 
+Notes about the command format:
+* Words in UPPER_CASE are the parameters to be supplied by the user.
+e.g. in `add n/NAME`, NAME is a parameter which can be used as: `add n/Assignment1`.
+* Parameters can be in any order.
+e.g. if the command specifies `n/NAME t/TIME`, `t/TIME n/NAME` is also acceptable.
+* Items in square brackets are optional. e.g `n/NAME  [d/DURATION]` can be used as `n/Task1 d/1` or as `n/Task1`.
+* For some commands, if no parameters are specified, the command will still execute in a default way.
+* For dates, the program follows the DD-MM-YYYY format.
+* For time, the program follows the HHMM format.
+
+### Adding a task: `add`
+Adds a task to the planner.
+
+Format: `add n/NAME t/TIME [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
+
+Parameters:
+* `NAME` is Name of the task.
+* `TIME` is Time to start doing this task (HHMM). For example, 1100.
+* `DURATION` is Expected time taken to complete task (in hours)
+   Default: 1 hour.
+* `DEADLINE` is Date to do task by (DD-MM-YYYY). For example, 16-09-2020.
+   Default: No deadline.
+* `RECURRENCE` is one of TODAY, MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY.
+   Default: TODAY.
+* `IMPORTANCE` is one of HIGH, MEDIUM, LOW.
+   Default: MEDIUM.
+* `ADDITIONAL-NOTES` is Notes of the task.
+   Default: No notes.
```

Comment:
`the name of the task` instead of `Name of the task`? And same for the other parameters.